sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/LoggingUtils.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.logging;
+
+import com.azure.core.util.CoreUtils;
+
+/**
+ * Contains utility methods for logging.
+ */
+public final class LoggingUtils {
+    private static final char CR = '\r';
+    private static final char LF = '\n';
+    
+    private LoggingUtils() {
+    }
+
+    /*
+     * Removes CRLF pattern in the {@code logMessage}.
+     *
+     * @param logMessage The log message to sanitize.
+     * @return The updated logMessage.
+     */
+    public static String sanitizeLogMessageInput(String logMessage) {
```

Comment:
few nits.
- the doc mentions CRLF but the code seems to remove any new line.
- consider calling this `removeNewLinesFromLogMessage` or so. Sanitization may also mean removing secrets (like keys) from the message and draw attention of people seeking such functionality here.